packages/ifixit-api-client/index.tsx

```diff
@@ -85,6 +85,7 @@ export class IFixitAPIClient {
                credentials: 'include',
                ...init,
                headers: headers,
+               cache: 'no-cache',
```

Comment:
Is this supposed to be no-store?
```suggestion
               cache: 'no-store',
```